lightning/src/ln/channelmanager.rs

```diff
@@ -8952,7 +8984,28 @@ macro_rules! create_refund_builder { ($self: ident, $builder: ty) => {
 		let context = OffersContext::OutboundPayment { payment_id, nonce, hmac: None };
 		let path = $self.create_blinded_paths_using_absolute_expiry(context, Some(absolute_expiry))
 			.and_then(|paths| paths.into_iter().next().ok_or(()))
-			.map_err(|_| Bolt12SemanticError::MissingPaths)?;
+			.map_err(|()| Bolt12RequestError::BlindedPathCreationFailed)?;
+
+		let total_liquidity: u64 = {
```

Comment:
@jkczyz 

Should we add a similar liquidity check for inbound liquidity? This would be helpful when we are creating an offer, or calling `request_refund_payment`.

Reply:
Offers can be long-lived and be paid more than once. So it would be only be accurate for short-lived, single-use offers. For those and refund payment request, I wonder if that approach is preferable as it allows us to give early indication of a failure to the user. Drawback is it would prevent creating an offer / sending an invoice even if liquidity  is soon-but-not-yet available (e.g., future JIT channel work). @tnull Thoughts?

Reply:
So IIUC, the outbound liquidity check for refunds makes sense since they’re likely to be claimed quickly. But for inbound liquidity, JIT channels could complicate things. Should we hold off on adding an inbound liquidity check in this PR? I’d appreciate any thoughts on how to proceed!

Reply:
Yeah, let's hold off.